backend/src/graphql_api/stable_coin.rs

```diff
@@ -166,6 +169,16 @@ impl StableCoin {
     }
 }
 
+#[derive(Parser)]
+struct App {
+    #[arg(
+        long = "node",
+        env = "CCDSCAN_INDEXER_GRPC_ENDPOINTS",
```

Comment:
One general design decision when creating the `indexer` service and the `graphQL API` service was that we decided to NOT have a node connection in the `graphQL API` service. That decision has pros/cons. 

One of the advantages was that it made it easier for us (devops) to run several instances of the GraphQL API services, without having dedicated nodes attached to them (no need to manage nodes for these services).
A disadvantage is that the data you need here would need to be collected by the indexer at every `createPLT` tx event and a table in postgres would collect all `tokenInfo` to provide it to the `grpahQL API` service.

Let me discuss that with the team and I will come back to you, but we might want to keep or break that design decision.